src/components/CloseOrderBtn.js

```diff
@@ -21,8 +18,12 @@ class CloseOrderBtn extends Component {
 
 CloseOrderBtn.propTypes = {
   shouldBeDisabled: PropTypes.bool.isRequired,
-  handleClick: PropTypes.func.isRequired
+  handleClick: PropTypes.func.isRequired,
+  text: PropTypes.string
 };
 
-export default CloseOrderBtn;
+CloseOrderBtn.defaultProps = {
+  text: 'Close Order'
```

Comment:
How we usually do it - set the static content as the second argument when creating the dom element. I like this way better way though, setting static content via a props.